README.md

```diff
@@ -0,0 +1,98 @@
+# ss18_showmaster
+Roboy wishes to be more entertaining and impressive. With his new skills he is reliably able to attract people to its presence independent to their relation to him. He appeals as well to an audience as to a group of developers, satisfying both of their preferences.
+
+
+## About
+This repository consists of four sections to make Roboy more entertaining on fairs.
+1. Facial Expressions to show more emotions
+2. Generative Model for Chitchat
+3. Vision Module
+4. Additions to Dialog for playing games
+
+Roboy Showmaster team wants to combine these sections to have an immersive and entertaining conversation.
+
+
+## How it works
+
+
+### 1. Facial Expressions
+In addition to the old faces and expressions in Unity, the following faces & emotions were added by animating them in Unity and and can be triggered via ROS:
+ - suprised Roboy
+ - crying Roboy
+ - irritated Roboy
+ - Roboy wearing sunglasses
+ 
+ The animations can be triggered via the code using Microsoft Visual Studies in RoboyUnityFace/Assets/RoboyAnimator.cs 
+ The corresponding computers must be connected using the same IP-address. Emotions can now be triggered via ROS. 
+
+### 2. Generative Models
+The old Generative Model was neither a state of the art implementation nor did it use a state of the art dataset as a basis. The new model is originating from Facebooks Dialog Research Platform [ParlAI](https://github.com/facebookresearch/ParlAI/). There, several state of the art implementations as well as several datasets are integrated in one platform. For Roboy, the profilememory implementation was choosen based on the Personachat dataset. 
+
+### 3. Vision Module
+The ROS Service Server receives service calls from the dialog as Strings that hold the desired filter. The vision module streams the video information from Roboy's camera to an iPad and applies the requested filter to the detected faces. The algorithm uses OpenCV and dlib library finding 68 facial landmarks to detect the most important face charateristics and to estimate the tilt angle of the face. The required filter mask is then applied automatically in-face superposition in real time. Following filters have been implemented: Roboy mask, mustache, pixelated sunglasses, flies, hat, crown and rainbow.
+
+### 4. Dialog Games
+The Dialog implements two games: 
```

Comment:
dialog doesnt implement the games...

"two new states for the Roboy dialog system (link to dialog system) were implemented:"